music21/harmony.py

```diff
@@ -2499,6 +2500,57 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBass(Harmony):
+    '''
+    The FiguredBassIndication objects store information about thorough bass figures.
+    It is created as a representation for <fb> tags in MEI and <figured-bass> tags in MusicXML.
+    The FiguredBassIndication object derives from the Harmony object and can be used
+    in the following way:
+    >>> fb = harmony.FiguredBass('#,6#', correspondingPart='P1')
+    >>> fb
+    <FiguredBass figures: #,6# correspondingPart: P1>
+
+    The single figures are stored as figuredBass.notation.Figure objects:
+    >>> fb.figNotation.figures[0]
+    <music21.figuredBass.notation.Figure 3 <Modifier # sharp>>
+    '''
+
+
+    def __init__(self,
+                 figureString: str | list[str] | None = None,
+                 correspondingPart: str | None=None,
+                 **keywords):
+        super().__init__(**keywords)
+
+        self.isFigure: bool = True
+        self.corresPart: str | None = None
+        self._figs: str = ''
+
+        if figureString:
+            if isinstance(figureString, list):
+                self._figs = ','.join(figureString)
+            elif isinstance(figureString, str):
+                if ',' in figureString:
+                    self._figs = figureString
+                else:
+                    self._figs = ','.join(figureString)
+        else:
+            self._figs = ''
+        self._figNotation = notation.Notation(self._figs)
+        self.corresPart = correspondingPart
+
+    @property
+    def figNotation(self) -> notation.Notation:
+        return self._figNotation
+
+    @figNotation.setter
+    def figNotation(self, figs: str | list[str] | None):
+        self._figNotation = notation.Notation(figs)
+
+    def __repr__(self):
+        return f'<{self.__class__.__name__} figures: {self.figNotation.notationColumn} correspondingPart: {self.corresPart}>'
```

Comment:
Music21Objects don't define `__repr__` just `_reprInternal`, which would be just the part from `figures: {...}` to the end before `>` -- let the Music21Object `__repr__` take care of the class name, etc.

Generally the representation just provides the most relevant information, so I think it could just be `return self.notation.notationColumn`.  in .show('text') it'll be obvious what Part the FiguredBass object belongs to.